certbot-dns-sakuracloud/certbot_dns_sakuracloud/__init__.py

```diff
@@ -0,0 +1,86 @@
+"""
+The `~certbot_dns_sakuracloud.dns_sakuracloud` plugin automates the process of completing
+a ``dns-01`` challenge (`~acme.challenges.DNS01`) by creating, and subsequently
+removing, TXT records using the Sakura Cloud DNS API.
+
+
+Named Arguments
+---------------
+
+========================================  =====================================
+``--dns-sakuracloud-credentials``         Sakura Cloud credentials_ INI file.
+                                          (Required)
+``--dns-sakuracloud-propagation-seconds`` The number of seconds to wait for DNS
```

Comment:
There's a formatting error in this table; this line extends past the edge of the column. You'll just need to tweak the number of "=" characters on the top and bottom lines then adjust the spacing of the other lines.

To test this when you're done, you can run `make html` from the `certbot-dns-sakuracloud/docs` directory (while your `venv` is activated.

---

@bmw: Perhaps we should add an automatic check of some sort that tries to generate the documentation for each package?

Reply:
Thanks!

I fixed and tested with `make html`.